src/test/java/core/basesyntax/service/impl/StorageServiceImplTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.StorageService;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class StorageServiceImplTest {
+    private StorageService storageServiceAdd = new StorageServiceAddImpl(new FruitDaoImpl());
+    private StorageService storageServicePurchase
+            = new StorageServicePurchaseImpl(new FruitDaoImpl());
+    private StorageService storageServiceReturn = new StorageServiceReturnImpl(new FruitDaoImpl());
+    private StorageService storageServiceSupply = new StorageServiceSupplyImpl(new FruitDaoImpl());
+    private ReportCreator reportCreator = new CreateReportImpl();
+
+    @Test
+    public void addFruitInStorage_Ok() {
+        Storage.fruits.clear();
+        storageServiceAdd.operateSupply("kiwi",25);
+        assertEquals("kiwi",String.valueOf(Storage.fruits.get(0).getNameFruit()));
+    }
+
+    @Test
+    public void addFruitInStorage() {
+        Storage.fruits.clear();
```

Comment:
to avoid duplication you can use a method if @After annotation. And storage will be cleared after each method

Reply:
Thank's. Used it.